src/services/connextService.ts

```diff
@@ -0,0 +1,96 @@
+import { getPoolFeeForUniV3, getXCallCallData, prepareSwapAndXCall } from '@connext/chain-abstraction';
+import { DestinationCallDataParams, Swapper, SwapAndXCallParams } from '@connext/chain-abstraction/dist/types';
+import { SdkConfig, create } from '@connext/sdk';
+import { RAW_NETWORKS, SUPPORTED_CHAINS_BY_CONNEXT } from 'config';
+import WalletService from './walletService';
+
+interface DomainID {
+  [key: number]: string;
+}
+
+export default class ConnextService {
+  sdkConfig: SdkConfig;
+
+  walletService: WalletService;
+
+  destinantionChainID: number;
+
+  constructor(walletService: WalletService, destinantionChainID: number) {
+    this.walletService = walletService;
+    this.destinantionChainID = destinantionChainID;
+  }
+
+  getRPCURL(chainID: number) {
+    const network = Object.values(RAW_NETWORKS).find((net) => net.chainId === chainID);
```

Comment:
No need to use RAW_NETWORKS. The `NETWORKS`  object will be more complete since its also hydrated from our SDK

Reply:
Sure